### DISCUSSION TITLE
tutorials/making-games-platformer

### DISCUSSION THREAD

**Discussion by giscus** [ct-js/docs.ctjs.rocks#142] [Pages' discussions]
# tutorials/making-games-platformer

Making Games: Platformer In this tutorial, we will create a small platformer with diamonds, checkpoints, moving platforms, and traps! You will learn how to detect collisions, us...

https://docs.ctjs.rocks/tutorials/making-games-platformer.html

**Comment by bl4nkspace** [comment]
very good tutorial ! 👏👏
